simplified-app-shared/src/main/java/org/nypl/simplified/app/reader/ReaderActivity.java

```diff
@@ -546,25 +560,18 @@ else if (id == 1) {
   }
 
   @Override public void onEPUBLoadSucceeded(
-    final Container c)
-  {
+    final Container c) {
     this.epub_container = c;
     final Package p = NullCheck.notNull(c.getDefaultPackage());
 
     final TextView in_title_text = NullCheck.notNull(this.view_title_text);
     UIThread.runOnUIThread(
-      new Runnable()
-      {
-        @Override public void run()
-        {
-          in_title_text.setText(NullCheck.notNull(p.getTitle()));
-        }
-      });
-
-
-
-    //TODO set package to sync manager.. this.syncManager.package = p;
-
+        new Runnable() {
+          @Override
+          public void run() {
+            in_title_text.setText(NullCheck.notNull(p.getTitle()));
```

Comment:
it looks like `notNull` raises if null is passed, how sure are we that this `package` has a title? 

These `NullCheckException` do not appear to be handled anywhere in this method, is that okay?

https://github.com/io7m/jnull/blob/04808485662d04bcab4534c4c3c1da6fb55e42be/com.io7m.jnull.core/src/main/java/com/io7m/jnull/NullCheck.java#L68

Reply:
It's by design to raise an uncaught exception when using `notNull()`